polynote-kernel/src/main/scala/polynote/kernel/LocalKernel.scala

```diff
@@ -117,6 +117,25 @@ class LocalKernel private[kernel] (
     } yield signatures
   }.catchAll(_ => ZIO.succeed(None))
 
+  override def goToDefinition(source: Either[String, CellID], pos: Int): TaskC[(List[DefinitionLocation], Option[String])] =
+    source match {
+      case Right(id) =>
+        val lookup = for {
+          (cell, interp, state) <- cellInterpreter(id, forceStart = true).onError(cause => Logging.error(cause))
+          locations             <- interp.goToDefinition(cell.content.toString, pos, state)
+        } yield locations
+        lookup.catchAll(_ => ZIO.succeed((Nil, None)))
```

Comment:
I see we did this in the `Left` case, any reason not to do it here as well?
```suggestion
        lookup.onError(Logging.error).catchAll(_ => ZIO.succeed((Nil, None)))
```